ocdskingfishersummarize/db.py

```diff
@@ -61,7 +61,11 @@ def format(self, statement, **kwargs):
             if isinstance(value, sql.Composable):
                 objects[key] = value
             elif isinstance(value, list):
-                objects[key] = sql.SQL(', ').join(sql.Identifier(entry) for entry in value)
+                entries = []
+                for entry in value:
+                    entries.append(sql.SQL('.').join([sql.Identifier(part) for part in entry.split(".")]))
+
+                objects[key] = sql.SQL(', ').join(entries)
```

Comment:
See https://www.psycopg.org/docs/sql.html#psycopg2.sql.Identifier

I'll actually make a commit to extract a method from here, so that instead of passing strings with dots we just pass the tuple arguments to Identifier directly.

```suggestion
                objects[key] = sql.SQL(', ').join(*sql.Identifier(entry.split('.')) for entry in value)
```